include/inviwo/core/interaction/trackball.h

```diff
@@ -161,21 +161,21 @@ class IVW_CORE_API Trackball : public CompositeProperty {
     OptionPropertyInt trackballMethod_;  /// Chooses which trackball method to use (mouse only,
                                          /// touch always follows finger)
     FloatProperty sensitivity_;          /// Controls the rotation sensitivity
-    FloatProperty verticalAngleLimit_;   /// Limits the angle between world up and view direction
-                                         /// when fixUp is True
     FloatProperty movementSpeed_;
     BoolProperty fixUp_;               /// Fixes the up vector to world_up in all rotation methods
     OptionPropertyInt worldUp_;        /// Defines which axis is considered up in world space
     FloatVec3Property customWorldUp_;  /// The custom world up direction (normalized)
+    FloatProperty verticalAngleLimit_;   /// Limits the angle between world up and view direction
+                                         /// when fixUp is True
 
     // Interaction restrictions
     BoolProperty handleInteractionEvents_;
     // Options to restrict translation along view-space axes.
     BoolProperty allowHorizontalPanning_;  ///< Enable/disable horizontal panning
     BoolProperty allowVerticalPanning_;    ///< Enable/disable vertical panning
     BoolProperty allowZooming_;            ///< Enable/disable zooming
+    BoolProperty allowWheelZooming_;       ///< Enable/disable zooming using the mouse wheel
 
-    FloatProperty maxZoomInDistance_;  ///< Cannot zoom in closer than this distance
```

Comment:
@martinfalk Any particular reason this functionality was removed?

Reply:
a) this property was actually never added (except for the copy constructor)
b) it was initialized with `0`
c) no way to change it